### ISSUE TITLE
Brother

### ISSUE THREAD

**Issue by Doctormap1** [vaibhavmagon/Python-Spam-Ham-SMS#5]
What do I need for sms spamming

**Comment by vaibhavmagon** [comment]
If you look here - https://github.com/vaibhavmagon/Python-Spam-Ham-SMS/blob/master/spam-real.py - there are 2 pieces of lines that can be replaced with streaming/s3 bucket data which can fetch realtime data and model can help and get a result in terms of spam or now.

# The data dump that helps classifier find out.
messages = pd.read_csv('data/spam-ham.csv', delimiter=',\t', names=['label', 'sms'], engine='python')

#The string that you want to check
messages_real = pd.read_csv('data/spam-real.csv', names=['sms'])

**Comment by Doctormap1** [comment]
Thanks boss I need chase lead and smtp sir boss please

**Comment by vaibhavmagon** [comment]
Sorry, didn't get you. Please be more elaborate.

**Comment by Doctormap1** [comment]
Yup boss I’m try to tell you that I need sms leads and  chase scam page 📄

**Comment by Doctormap1** [comment]
Boss I hope I can run sms spamming with my phone
